searchcore/src/vespa/searchcore/proton/common/handlermap.hpp

```diff
@@ -115,6 +113,20 @@ class HandlerMap {
         return HandlerSP();
     }
 
+    /**
+     * Returns the handler for the given document type. If no handler was
+     * registered, this method returns an empty shared pointer.
```

Comment:
nullptr instead of empty shared pointer ?

Reply:
Fixed